src/components/GetStarted.tsx

```diff
@@ -4,7 +4,7 @@ import { DownloadImage } from './DownloadImage';
 import { Indicator } from './Indicator';
 import { ExternalLink } from './ExternalLink';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faArrowToBottom } from '@fortawesome/pro-regular-svg-icons';
+import { faDownload } from '@fortawesome/free-solid-svg-icons';
```

Comment:
need to add this to package.json

Reply:
Will do, it uses it from rendition currently